src/test/java/org/broadinstitute/hellbender/engine/filters/ReadFilterPluginUnitTest.java

```diff
@@ -185,6 +185,24 @@ public void testEnableDuplicateFilter(final String[] arguments) {
         clp.parseArguments(System.out, arguments);
     }
 
+    @DataProvider(name = "duplicateDisabledFilters")
+    public Object[][] duplicateDisabledFilters() {
+        return new Object[][] {
+                {Collections.singletonList(ReadFilterLibrary.MAPPED),
+                        new String[]{
+                                "--DF", ReadFilterLibrary.MAPPED.getClass().getSimpleName(),
+                                "--DF", ReadFilterLibrary.MAPPED.getClass().getSimpleName()}
+                }
+        };
+    }
+
+    @Test(dataProvider = "duplicateDisabledFilters", expectedExceptions = CommandLineException.BadArgumentValue.class)
+    public void testDisableDuplicateFilter(final List<ReadFilter> defaults, final String[] arguments) {
+        CommandLineParser clp = new CommandLineArgumentParser(new Object(),
+                Collections.singletonList(new GATKReadFilterPluginDescriptor(defaults)));
+        clp.parseArguments(System.out, arguments);
```

Comment:
Tests shouldn't print to standard out, since we can't control this through the test verbosity, and we have log file limits on, eg., travis. Can we use something like `NullOutputStream` in apache commons?